app/views/categories/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= @category.name %></h1>
```

Comment:
Also, it appears, that the "ask a question" button is missing now. I can't find it on the homepage and on this page.